Add exit validator option

tropicar opened the pull request:
With this changes the user can do a voluntary exit of their validators, he has to write on 2 env vars:
EXIT_VALIDATOR: I want to exit my validators  he must write exactly that phrase
KEYSTORES_VOLUNTARY_EXIT: In this one, the user would write the keystores in the next format: 0x849384938,0x83948398

tropicar committed:
env vars for exit validators

tropicar committed:
Check vars and exit command

The user must write i want to exit my validators and the keystores in the keystores_voluntary_exit
The format for this value is:
0x898493....,0x84938493..

pablomendezroyo commented:
LGTM! Has this been tested?

pablomendezroyo commented on validator/entrypoint.sh:
```diff
@@ -5,6 +5,22 @@ NETWORK="mainnet"
 WEB3SIGNER_API="http://web3signer.web3signer.dappnode:9000"
 
+
+if [[ "$EXIT_VALIDATOR" == "I want to exit my validators" ]]; then
```
what would happen once the user has set this value to the env? it will attempt to exit the validators even if there are new imports in the future, which is really dangerous.

pablomendezroyo replied:
We should be safed since we are defining the validators to exit in the variable `KEYSTORES_VOLUNTARY_EXIT`

tropicar replied:
Not at all, the user MUST define the public keys in the KEYSTORES_VOLUNTARY_EXIT , but I think it would be a good practice to check that must have content on the KEYSTORES_VOLUNTARY_EXIT.

tropicar committed:
Add checks and improve logs

Now its mandatory to define th evar of the public key to enter in the condition of do the exit

tropicar commented:
> LGTM! Has this been tested?

Now the user must define the public key, and if th value is not correct , the command fails and not do anything
```
Check connectivity with the web3signer
Proceeds to do the voluntary exit of the next keystores:
example
Loading configuration...
Invalid configuration. Signer public key is invalid
```
